include/_antic.h

```diff
@@ -57,42 +57,42 @@ struct __antic {
-#define DL_CHR40x8x4    4       /* colour, 40 character & 8 scanlines per mode line (GR. 12) */
-#define DL_CHR40x16x4   5       /* colour, 40 character & 16 scanlines per mode line (GR. 13) */
-#define DL_CHR20x8x2    6       /* colour (duochrome per character), 20 character & 8 scanlines per mode line (GR. 1) */
-#define DL_CHR20x16x2   7       /* colour (duochrome per character), 20 character & 16 scanlines per mode line (GR. 2) */
+#define DL_CHR40x8x1    (unsigned char) 2       /* monochrome, 40 character & 8 scanlines per mode line (GR. 0) */
+#define DL_CHR40x10x1   (unsigned char) 3       /* monochrome, 40 character & 10 scanlines per mode line */
+#define DL_CHR40x8x4    (unsigned char) 4       /* colour, 40 character & 8 scanlines per mode line (GR. 12) */
+#define DL_CHR40x16x4   (unsigned char) 5       /* colour, 40 character & 16 scanlines per mode line (GR. 13) */
+#define DL_CHR20x8x2    (unsigned char) 6       /* colour (duochrome per character), 20 character & 8 scanlines per mode line (GR. 1) */
+#define DL_CHR20x16x2   (unsigned char) 7       /* colour (duochrome per character), 20 character & 16 scanlines per mode line (GR. 2) */
 
-#define DL_MAP40x8x4    8       /* colour, 40 pixel & 8 scanlines per mode line (GR. 3) */
-#define DL_MAP80x4x2    9       /* 'duochrome', 80 pixel & 4 scanlines per mode line (GR.4) */
-#define DL_MAP80x4x4    10      /* colour, 80 pixel & 4 scanlines per mode line (GR.5) */
-#define DL_MAP160x2x2   11      /* 'duochrome', 160 pixel & 2 scanlines per mode line (GR.6) */
-#define DL_MAP160x1x2   12      /* 'duochrome', 160 pixel & 1 scanline per mode line (GR.14) */
-#define DL_MAP160x2x4   13      /* 4 colours, 160 pixel & 2 scanlines per mode line (GR.7) */
-#define DL_MAP160x1x4   14      /* 4 colours, 160 pixel & 1 scanline per mode line (GR.15) */
-#define DL_MAP320x1x1   15      /* monochrome, 320 pixel & 1 scanline per mode line (GR.8) */
+#define DL_MAP40x8x4    (unsigned char) 8       /* colour, 40 pixel & 8 scanlines per mode line (GR. 3) */
+#define DL_MAP80x4x2    (unsigned char) 9       /* 'duochrome', 80 pixel & 4 scanlines per mode line (GR.4) */
+#define DL_MAP80x4x4    (unsigned char) 10      /* colour, 80 pixel & 4 scanlines per mode line (GR.5) */
+#define DL_MAP160x2x2   (unsigned char) 11      /* 'duochrome', 160 pixel & 2 scanlines per mode line (GR.6) */
+#define DL_MAP160x1x2   (unsigned char) 12      /* 'duochrome', 160 pixel & 1 scanline per mode line (GR.14) */
+#define DL_MAP160x2x4   (unsigned char) 13      /* 4 colours, 160 pixel & 2 scanlines per mode line (GR.7) */
+#define DL_MAP160x1x4   (unsigned char) 14      /* 4 colours, 160 pixel & 1 scanline per mode line (GR.15) */
+#define DL_MAP320x1x1   (unsigned char) 15      /* monochrome, 320 pixel & 1 scanline per mode line (GR.8) */
 
 /* modifiers on mode lines */
-#define DL_HSCROL       16
-#define DL_VSCROL       32
-#define DL_LMS          64
+#define DL_HSCROL(x)    (unsigned char)((x) | 16)
```

Comment:
Sorry that you don't appreciate the change, but here my final thoughts regarding the issue (I have no problem to cancel the pull request (it would still live in my fork) - it seems it happens to often that someone comes up with a little change request supplying a test and documentation too.. ;^) ):

Creating a display list on the C side isn't very comfortable. When using the unsigned char array approach, you aren't able to initialise the screen addresses, as all operations on addresses to extract a high- and a low byte are non-const in cc65 (e.g.  adr >> 8,  adr & 0xFF, (unsigned char)addr etc.). Only solution was to set up these addresses in C code (needs memory, takes performance).
AFAIK the only solution consists in using the void-data feature of cc65, requiring unsigned char data type for ANTIC instructions, otherwise you would go the uncomfortable route of writing:

void DisplayList = {
    (unsigned char) DL_BLK8,
    (unsigned char) DL_BLK8,
    (unsigned char) DL_BLK8,
    (unsigned char) (DL_CHR20x8x2 | DL_LMS),
    screen,
    (unsigned char) DL_CHR20x8x2,
    (unsigned char) DL_CHR20x8x2,
    (unsigned char) DL_CHR20x8x2,
    (unsigned char) DL_BLK4,
    (unsigned char) DL_CHR20x8x2,
    (unsigned char) DL_JVB
};

Not be able to initialise the addresses in a C display list forced at least me to do so on the assembler side. Only for that purpose I've often created an assembler file, which seems much more unfriendly for newbies to me than offering a neat solution for C.

Due to the documentation level, the reason I mentioned before and overall usage of cc65 on the Atari I doubt that the little change on modifier level implies many source files to break. I think ease the use (which this change IMHO does) is quite accepted.
BTW: Nobody seemed to care that a change once broke my SMC macros for ca65 which is even part of the suite: https://sourceforge.net/p/cc65/mailman/message/35225776/
I regard that anyway as "OK", since I also think that this is a rarely used feature.

Since this discussion is likely to exceed the creation time for the change (and of course my spare time I willing to spend to contribute), I have unfortunately have to say take it or leave it.